server/routes/index.ts

```diff
@@ -33,7 +32,19 @@ export const get = (router: Router, path: string, handler: RequestHandler): Rout
 export const post = (router: Router, path: string, handler: RequestHandler): Router =>
   router.post(path, asyncMiddleware(handler))
 
-export default function routes(router: Router, services: Services): Router {
+export default function routes(
+  router: Router,
+  services: {
+    interventionsService: InterventionsService
+    ramDeliusApiService: MockRamDeliusApiService | RamDeliusApiService
```

Comment:
Why are you defining it individually instead of the old code `services: Services`